crates/swc_ecma_minifier/src/compress/pure/strings.rs

```diff
@@ -126,64 +126,73 @@ impl Pure<'_> {
         };
         let mut cur_cooked_str = Wtf8Buf::new();
         let mut cur_raw_str = String::new();
+        let mut q_iter = tpl.quasis.take().into_iter();
+        let e_iter = tpl.exprs.take().into_iter();
+
+        // Consume a quasis first to make sure it align with exprs
+        // quasis.len() == exprs.len() + 1
+        if let Some(q) = q_iter.next() {
+            cur_cooked_str.push_wtf8(&q.cooked.unwrap());
```

Comment:
This may panic

Reply:
I think we can assume cooked always not `None` here, but anyway, I can fallback to from_tpl_raw here.